src/general/OptionsMenu.cs

```diff
@@ -54,6 +56,7 @@ public class OptionsMenu : Control
     // Performance tab
     private OptionButton cloudInterval;
     private OptionButton cloudResolution;
+    private OptionButton msaaResolution;
```

Comment:
The MSAA amount should be in the graphics tab, as that's where all games put their anti-aliasing options.